tests/unit/conftest.py

```diff
@@ -80,3 +81,33 @@ def teardown():
 
     with app.app_context():
         yield app
+
+
+@pytest.yield_fixture(scope='session', autouse=True)
+def db_app(request):
```

Comment:
This is a use case I hand't anticipated. I'd say to create this in `tests/integration/conftest.py`, and copy from the `app` fixture only the lines that create the table. What is that needs the tables to be there, but not necessarily the records?